src/controllers/trip.controller.js

```diff
@@ -39,27 +39,42 @@ class tripController {
       const status = 'pending';
       const { type } = req.body;
 
-      const createdTrip = await tripService.CreateTrip(req, req.body, tripid, type);
+      const createdTrip = await tripService.CreateTrip(
+        req,
+        req.body,
+        tripid,
+        type
+      );
 
       const { tripId } = createdTrip;
 
       const userId = req.user.id;
 
-
       const userManager = await userService.findUserManager(userId);
       const { managerId } = userManager[0];
       const requestData = {
-        tripId, userId, managerId, status
+        tripId,
+        userId,
+        managerId,
+        status
       };
       const tripRequest = await tripService.CreateTripRequest(requestData);
-      await NotificationService.sendNotification('trip_request_event', managerId, `New ${type} trip request.`, `${req.user.firstName} has requested a new ${type}`, tripRequest.id, 'http://localhost:3000/api/v1/notifications');
-      return response.successMessage(res, 'Trip created successfully', 201, tripRequest.status);
-    } catch (e) {
-      return response.errorMessage(
+      await NotificationService.sendNotification(
+        'trip_request_event',
+        managerId,
+        `New ${type} trip request.`,
+        `${req.user.firstName} has requested a new ${type}`,
+        tripRequest.id,
+        'http://localhost:3000/api/v1/notifications'
+      );
+      return response.successMessage(
```

Comment:
Unnecessary change. I think we might have the previous code we don't need to change it ?